homeassistant/components/totalconnect/config_flow.py

```diff
@@ -0,0 +1,60 @@
+"""Config flow for the Total Connect component."""
+import logging
+
+from total_connect_client import TotalConnectClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import CONF_PASSWORD, CONF_USERNAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class TotalConnectConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Total Connect config flow."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initiated by the user."""
+        errors = {}
+
+        if user_input is not None:
+            # Validate user input
+            username = user_input[CONF_USERNAME]
+            password = user_input[CONF_PASSWORD]
+
+            await self.async_set_unique_id(username)
+            self._abort_if_unique_id_configured()
+
+            valid = await self.is_valid(username, password)
+
+            if valid:
+                # authentication success / valid
+                return self.async_create_entry(
+                    title="Total Connect",
```

Comment:
We always put the integration name in front of the title, so this should identify the account.
```suggestion
                    title=username,
```